src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -746,65 +748,68 @@ public static String processPageSourceForFailureCase(String content) {
+                    + "[\\w+-][\\w+!#$%&'*/=?^_`{}~-]*+(\\.[\\w+!#$%&'*/=?^_`{}~-]+)*+@([A-Za-z0-9-]+\\.)*[A-Za-z]+", "{*}")
+            //questionid
+            .replaceAll("([a-zA-Z0-9-_]){62,}","{*}")
+            //commentid
+            .replaceAll("\\\"([0-9]){16}\\\"", "\\\"{*}\\\"")
+            // comment div ids (added after standardization)
+            .replaceAll("responseCommentRow-[0-9]{16}", "responseCommentRow-{*}")
+            .replaceAll("commentBar-[0-9]{16}", "commentBar-{*}")
+            .replaceAll("plainCommentText-[0-9]{16}", "plainCommentText-{*}")
+            .replaceAll("commentdelete-[0-9]{16}", "commentdelete-{*}")
+            // tooltip style
+            .replaceAll("style=\"top: [0-9]{2,4}px; left: [0-9]{2,4}px; display: block;\"",
+                        "style=\"top: {*}px; left: {*}px; display: block;\"")                
+            //commentid in url
+            .replaceAll("#[0-9]{16}", "#{*}")
+            // the test accounts/ email
+            .replace(TestProperties.inst().TEST_STUDENT1_ACCOUNT, "${test.student1}")
+            .replace(TestProperties.inst().TEST_STUDENT2_ACCOUNT, "${test.student2}")
+            .replace(TestProperties.inst().TEST_INSTRUCTOR_ACCOUNT, "${test.instructor}")
+            .replace(TestProperties.inst().TEST_ADMIN_ACCOUNT, "${test.admin}")
+            .replace(TestProperties.inst().TEST_UNREG_ACCOUNT, "${test.unreg}")
+            .replace(Config.SUPPORT_EMAIL, "${support.email}")
+            // today's date
+            .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "{*}")
+            .replace(TimeHelper.formatDate(now), "{*}")
+            // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
+            .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "{*}")
+            // now (used in opening time/closing time Grace period)
+            .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "{*}")
+            // now (used in comments last edited date) e.g. [Thu May 07 07:52:13 UTC 2015]
+            .replaceAll(new SimpleDateFormat("EEE MMM dd ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC [0-9]{4}", "{*}")
+            // jQuery local
+            .replace("/js/lib/jquery.min.js", "{*}/jquery.min.js")
+            // jQuery CDN
+            .replace("https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js", "{*}/jquery.min.js")
+            // jQuery-ui local
+            .replace("/js/lib/jquery-ui.min.js", "{*}/jquery-ui.min.js")
+            // jQuery-ui CDN
+            //<script[^>]*>(.*?)</script>
+            .replace("https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js", "{*}/jquery-ui.min.js");
+        Pattern p = Pattern.compile("<span class=\"submissionsNumber\".*?</span>", Pattern.DOTALL);
```

Comment:
Here is the meaty part, the rest are diff-ed by indentation changes since return is moved down here, can ignore everything above Pattern p.